sampler.go

```diff
@@ -361,18 +361,23 @@ func (s *AdaptiveSampler) IsSampled(id TraceID, operation string) (bool, []Tag)
 	return false, nil
 }
 
+func (s *AdaptiveSampler) trySampling(span *Span, operationName string) (sampled bool, tags []Tag) {
```

Comment:
nit: I don't see these named return values frequently in Jaeger. Why use them here?

Reply:
It is technically idiomatic Go, makes the code shorter, and the signature self-documenting.

But I can change to vars.